erpnext/support/doctype/support_settings/support_settings.json

```diff
@@ -125,10 +133,67 @@
    "fieldname": "allow_resetting_service_level_agreement",
    "fieldtype": "Check",
    "label": "Allow Resetting Service Level Agreement"
+  },
+  {
+   "fieldname": "support_portal_setting_section",
+   "fieldtype": "Section Break",
+   "label": "Support Portal Settings",
+   "show_days": 1,
+   "show_seconds": 1
+  },
+  {
+   "default": "We are here to help",
+   "fieldname": "greeting_text",
+   "fieldtype": "Data",
+   "label": "Greeting Text",
+   "show_days": 1,
+   "show_seconds": 1
+  },
+  {
+   "default": "Left",
+   "fieldname": "greeting_text_and_search_bar_alignment",
+   "fieldtype": "Select",
+   "label": "Greeting Text and Search Bar Alignment",
+   "options": "Left\nCenter\nRight",
+   "show_days": 1,
+   "show_seconds": 1
+  },
+  {
+   "default": "Color",
+   "fieldname": "background",
+   "fieldtype": "Select",
+   "label": "Background",
+   "options": "Color\nImage",
+   "show_days": 1,
+   "show_seconds": 1
+  },
+  {
+   "default": "#ffffff",
+   "depends_on": "eval:doc.background==\"Color\"",
+   "fieldname": "select_color",
+   "fieldtype": "Color",
+   "label": "Select Color",
+   "show_days": 1,
+   "show_seconds": 1
+  },
+  {
+   "depends_on": "eval:doc.background==\"Image\"",
+   "fieldname": "add_image",
+   "fieldtype": "Attach Image",
+   "label": "Add Image",
+   "show_days": 1,
+   "show_seconds": 1
+  },
```

Comment:
No need for these at the moment

Reply:
Again as per the plan to give customization option, hence more option added in doctype

Reply:
Just keep the alignment option, not any morr